lib/misc/blocks.coffee

```diff
@@ -0,0 +1,56 @@
+# TODO: docstrings
+
+scopes = require './scopes'
+
+module.exports =
+
+  getLine: (ed, l) ->
+    ed.getTextInBufferRange [[l, 0], [l, Infinity]]
```

Comment:
[Yep](https://github.com/JunoLab/atom-ink/blob/master/lib/editor/result.coffee#L61). 

(I'll try to work through the rest of this soonish...)